### DISCUSSION TITLE
AWS SaaS Boost Office Hours

### DISCUSSION THREAD

**Discussion by brtrvn** [awslabs/aws-saas-boost#106] [Show and tell]
Join us for our live broadcast office hours on [Twitch](https://www.twitch.tv/aws)!

We're live weekly on Wednesday afternoons at 1PM PST/PDT.

Office hours are a place for us to dive deep on demos, code examples and experiments with the SaaS Boost community.

We now have a [Twitch Collection](https://www.twitch.tv/collections/1V3OevwQkRYsSw) for our past broadcasts. Simply mouse over the video title in the upper left of the player to see all videos in the collection.

Episode 1: [Installing and getting started with SaaS Boost](https://www.twitch.tv/videos/1084380760?collection=1V3OevwQkRYsSw)
Episode 2: [Tenant onboarding in depth](https://www.twitch.tv/videos/1084404670?collection=1V3OevwQkRYsSw)
Episode 3: [Enhance SaaS Boost with functions](https://www.twitch.tv/videos/1099760302?collection=1V3OevwQkRYsSw) (walkthru of building the PR to resolve Issue #70)
Episode 4: [Part 1 - Create a new extension for tenant environments](https://www.twitch.tv/videos/1100855281?collection=1V3OevwQkRYsSw)
Episode 5: [Part 2 - Create a new extension for tenant environments](https://www.twitch.tv/videos/1100925739?collection=1V3OevwQkRYsSw)
Episode 6: [Part 3 - Create a new extension for tenant environments](https://www.twitch.tv/videos/1107334894?collection=1V3OevwQkRYsSw)
Episode 7: [Part 4 - Create a new extension for tenant environments](https://www.twitch.tv/videos/1181529505?collection=1V3OevwQkRYsSw)
Episode 8: [Shared File System Deep Dive](https://www.twitch.tv/videos/1149684609)

**Comment by brtrvn** [comment]
Post your ideas for upcoming office hours broadcasts! What would you like to see? What would you like to discuss?

**Comment by brtrvn** [comment]
Twitch URLs updated. If you were getting 404's, try again and let us know if you can't reach the content. After a limited amount of time, you have to archive your Twitch broadcasts or they get dropped. Unfortunately this changes the URL and viewer count. We're investigating the best way to archive and serve this content for the community going forward.

**Reply by HatariOS** [reply]
The above links are not working:
 error:"Sorry. Unless you’ve got a time machine, that content is unavailable."

Episode 4: Part 1 - Create a new extension for tenant environments
Episode 5: Part 2 - Create a new extension for tenant environments

**Reply by brtrvn** [reply]
Thanks for the head's up. We now have a "collection" on Twitch which hopefully will make it easier to find past broadcasts. Links above are updated.

**Comment by brtrvn** [comment]
We wrapped up our series on how to add a new "extension" to SaaS Boost this week. Check out episodes 4-7 above to walk through all the gory details on adding support for DynamoDB to your tenant environments. We will be cleaning up the code you see in the videos and submitting a proper pull request in the coming weeks!

**Comment by brtrvn** [comment]
A fast overview and demo of SaaS Boost was presented as part of the Open@Amazon virtual conference. Here's a recording of that session: https://www.twitch.tv/videos/1059171908
